### DISCUSSION TITLE
Multiple tooltips for same tip

### DISCUSSION THREAD

**Discussion by rose-m** [JanDeDobbeleer/oh-my-posh#5075] [Ideas]
I really like the customizability and just switched over from OMZ - thanks for your great work!

One thing I was trying to get working was to have multiple tooltips show up for a single "tip", i.e. when I type `terraform` I'd like to see information for both AWS and Azure accounts. I've configured the tooltips for both of them and they work independently but for `terraform` only the last defined one actually gets triggered.

That matches with the logic I find here: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/src/engine/tooltip.go#L11
So it iterates over all tooltips and in the end just takes the last one that should be rendered for the given tip.

Would it be possible to switch this and basically render all tooltips? That would allow for a lot greater composition abilities 👍 

**Update:** I've created a PR: https://github.com/JanDeDobbeleer/oh-my-posh/pull/5076

**Comment by JanDeDobbeleer** [comment]
@rose-m awesome idea! I'm going to have a look at this over lunch!
